src/Sulu/Component/Content/Document/Subscriber/RedirectTypeSubscriber.php

```diff
@@ -28,7 +28,7 @@ public static function getSubscribedEvents()
     {
         return [
             Events::METADATA_LOAD => 'handleMetadataLoad',
-            Events::PERSIST => 'handlePersist',
+            Events::PERSIST => ['handlePersist', 15],
```

Comment:
could we handle this "dependencies" somehow in the document-manager? it would be quite cool to define this subscriber after this one (use service-ids or class-names) to be sure that it is also working after changing the priority? /cc @dantleech